### PR TITLE
TRUNK-5930: PersonName Domain - Switching from Hibernate Mappings to Annotations

### PR DESCRIPTION
**Pull request by Ndacyayisenga-droid**
## Issue I worked on

see https://issues.openmrs.org/browse/TRUNK-5930

cc @dkayiwa  @ibacher  @Ruhanga  @mseaton

### CODE REVIEW

File: api/src/test/java/org/openmrs/api/OrderServiceTest.java [hunk 1]
```diff
@@ -51,50 +50,13 @@
...
 import org.hibernate.cfg.Configuration;
-import org.junit.Before;
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Disabled;
 import org.junit.jupiter.api.Test;
 
-import org.openmrs.ConceptMapType;
+import org.openmrs.*;
 import org.openmrs.Order.Action;
-import org.openmrs.OrderAttribute;
```
Comment (sherrif10) [inline api/src/test/java/org/openmrs/api/OrderServiceTest.java:61]: How is this related to switching from Hibernate mappings to annotations ,was it an overview?

**Comment by jwnasambu** [comment]
@Ndacyayisenga-droid  kindly add a description of what you changed.
